### DISCUSSION TITLE
Is there an easy way to include session language in the schedule?

### DISCUSSION THREAD

**Discussion by shaib** [pretalx/pretalx#1807] [Ideas] [answered]
Hi,

Running a bilingual conference, it would be really helpful if we can show session languages in the schedule.

There's an old discussion #1202 which sets high standards for doing this right; but I'd rather have something -- even the notorious use of flags, or even just language names -- in the schedule; the session details are not enough.

Last year we achieved it by manually adding flag emojis as prefixes to session titles, which seems, to put it gently, sub-optimal.

Does anyone have a solution?

**Comment by rixx** [comment, marked as answer]
We should definitely offer *something* here, so I opened an issue for this at #2032.
